.github/workflows/docker-build.yml

```
@@ -0,0 +1,70 @@
+        push: true
+        platforms: linux/amd64,linux/arm64
+        build-args: |
+          NPROC=$(nproc)
+        cache-from: |
+          type=local,src=/tmp/.buildx-cache
+          type=registry,ref=fastcalosim/${{ matrix.os }}-fastcalosim:cache
+        cache-to: |
+          type=local,dest=/tmp/.buildx-cache
+          type=registry,ref=fastcalosim/${{ matrix.os }}-fastcalosim:cache,mode=max
+        tags: fastcalosim/${{ matrix.os }}-fastcalosim:latest
+        progress: plain
```

actionlint (1.7.4)

input "progress" is not defined in action "docker/build-push-action@v5". available inputs are "add-hosts", "allow", "annotations", "attests", "build-args", "build-contexts", "builder", "cache-from", "cache-to", "cgroup-parent", "context", "file", "github-token", "labels", "load", "network", "no-cache", "no-cache-filters", "outputs", "platforms", "provenance", "pull", "push", "sbom", "secret-envs", "secret-files", "secrets", "shm-size", "ssh", "tags", "target", "ulimit"

(action)